packages/hash/frontend/src/components/Popover.tsx

```diff
@@ -0,0 +1,14 @@
+import { Popover as MuiPopover, PopoverProps } from "@mui/material";
```

Comment:
We should ban importing the MUI Popover if we add this component using eslint, [just as we are doing in the BP repo](https://github.com/blockprotocol/blockprotocol/blob/a38326979c47a74e70c987ddf5f5c65cb2b29b01/.eslintrc.js#L250-L255)